chroot-bin/croutoncycle

```diff
@@ -101,42 +108,25 @@ if [ ! -f "/sys/class/tty/tty0/active" ]; then
         read -r freonowner < "/tmp/crouton-lock/display"
     fi
     freonowner="${freonowner:-0}"
-    winlist="aura*"
-    aurawin="aura"
     tty=''
 else
-    # List windows on :0. Includes aura
-    winlist="`host-x11 croutonwmtools list nim | \
-              sort | awk '{ printf $NF " " }'`"
-    aurawin="`host-x11 croutonwmtools list ni | \
-              awk '$1 == "aura_root_0" { print $NF; exit }'`"
     tty="`cat '/sys/class/tty/tty0/active'`"
 fi
 
-# Combine the two
-fulllist="$winlist$displist"
-fulllist="${fulllist% }"
-
+# Determine current display
 if [ "$freonowner" = 0 -o "$tty" = 'tty1' ]; then
-    # Either in Chromium OS, xephyr/xiwi chroot, or window.
-    # Active window is starred.
-    for disp in $winlist; do
-        if [ "${disp%"*"}" != "$disp" ]; then
-            curdisp="$disp"
-            if [ -n "$xiwiactive" -a "${disp%"*"}" = "$aurawin" -a \
-                    -s "$CRIATDISPLAY" ]; then
-                kiwidisp="`cat $CRIATDISPLAY`"
-                if [ "${kiwidisp#:[0-9]}" != "$kiwidisp" ]; then
-                    curdisp="$kiwidisp"
-                fi
-            fi
-            break
+    # In Chromium OS or xiwi chroot
+    curdisp='cros'
+    if [ -n "$xiwiactive" -a -s "$CRIATDISPLAY" ]; then
+        kiwidisp="`cat $CRIATDISPLAY`"
+        if [ "${kiwidisp#:[0-9]}" != "$kiwidisp" ]; then
+            curdisp="$kiwidisp"
         fi
-    done
+    fi
 elif [ -z "$freonowner" ]; then
     # Poll the displays to figure out which one owns this VT
     curdisp="$tty"
-    for disp in $displist; do
+    for disp in ${displist#cros}; do
```

Comment:
Can you separate the != cros test in the loop to make sure it does not bite us if we happen to change the order in displist?